knowhere/knowhere/index/vector_index/faiss_common/AuxIndexStructures.h

```diff
@@ -0,0 +1,258 @@
+/**
```

Comment:
This file is only used by IDMAP::QueryByDistance() and BinaryIDMAP::QueryByDistance(),
and these 2 APIs have no corresponding SDK.
Suggest to "#if 0" these 2 APIs, and we can not copy AuxIndexStructure.h